geometry_msgs/PoseStamped pose
std_msgs/Bool detected